scripts/pre_commit_linter.py

```diff
@@ -603,8 +608,11 @@ def _check_bad_patterns(all_files):
     total_error_count = 0
     summary_messages = []
     all_files = [
-        filename for filename in all_files if not
-        any(fnmatch.fnmatch(filename, pattern) for pattern in EXCLUDED_PATHS)]
+        filename for filename in all_files if not (
+            any(
```

Comment:
Too many parens and indentation, I think. Hard to read.

Consider:

```
    ... if not (
        filename.endswith('pre_commit_linter.py') or
        any(
            fnmatch.fnmatch(filename, pattern)
            for pattern in EXCLUDED_PATHS)
    )]
```

Reply:
Done.